include/inviwo/core/properties/directoryproperty.h

```diff
@@ -55,6 +55,8 @@ class IVW_CORE_API DirectoryProperty : public FileProperty {
     DirectoryProperty& operator=(const DirectoryProperty& that);
     virtual DirectoryProperty* clone() const override;
 
+    virtual void requestFile() override;
```

Comment:
Add documentation. Should be requestDirectory?

Reply:
Since DirectoryProperty inherit from FileProperty I wanted to override the function that FileProperty runs if the property is a DirectoryProperty. I don't know how a requestDirectory function would get called if such a function was added.

Reply:
Ah, I see, you should add the corresponding documentation of the FileProperty::requestFile() 
In other words that it will present a dialog to choose directory and its path will be set